src/Tizen.Wearable.CircularUI.Forms/GoogleMapView.cs

```diff
@@ -0,0 +1,206 @@
+﻿/*
+ * Copyright (c) 2018 Samsung Electronics Co., Ltd All Rights Reserved
+ *
+ * Licensed under the Flora License, Version 1.1 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://floralicense.org/license/
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.Collections.Specialized;
+using System.ComponentModel;
+using System.Linq;
+using Xamarin.Forms;
+using Xamarin.Forms.Maps;
+
+namespace Tizen.Wearable.CircularUI.Forms
+{
+    /// <summary>
+    /// The MapView class is used to display a map on the screen.
+    /// </summary>
+    public class GoogleMapView : View, IGoogleMapViewController
+    {
+        /// <summary>
+        /// BindableProperty. Identifies the ItemsSource bindable property.
+        /// </summary>
+        public static readonly BindableProperty ItemsSourceProperty = BindableProperty.Create(nameof(IEnumerable), typeof(IEnumerable), typeof(GoogleMapView),
+            default(IEnumerable), propertyChanged: (b, o, n) => ((GoogleMapView)b).OnItemsSourcePropertyChanged((IEnumerable)o, (IEnumerable)n));
+
+        /// <summary>
+        /// BindableProperty. Identifies the ItemTemplate bindable property.
+        /// </summary>
+        public static readonly BindableProperty ItemTemplateProperty = BindableProperty.Create(nameof(ItemTemplate), typeof(DataTemplate), typeof(GoogleMapView),
+            default(DataTemplate), propertyChanged: (b, o, n) => ((GoogleMapView)b).OnItemTemplatePropertyChanged((DataTemplate)o, (DataTemplate)n));
+
+        readonly ObservableCollection<Pin> _pins = new ObservableCollection<Pin>();
+
+        private GoogleMapOption _option;
+
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public event EventHandler LoadMapRequested;
+
+        public GoogleMapView()
+        {
+            var option = new GoogleMapOption(new Position(41.890202, 12.492049));
+            MapOption = option;
+        }
+
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        internal GoogleMapOption MapOption
+        {
+            get
+            {
+                return _option;
+            }
+            set
+            {
+                if (_option.Equals(value))
+                    return;
+                _option = value;
+
+            }
+        }
+
+        /// <summary>
+        /// An IList of the Pins on this MapView.
+        /// </summary>
+        public IList<Pin> Pins
```

Comment:
What happen if user set `Pin.Type`? and Can user use `Clicked` event for these Pin?